### PR TITLE
Daos rpm update

### PR DESCRIPTION
**Pull request by mchaarawi**
## Author Checklist
* [ ] Reference appropriate issues (with "Fixes" or "See" as appropriate)
* [ ] Remove xfail from the test suite when fixing a test
* [ ] Commits are self-contained and do not do two things at once
* [ ] Commit message is of the form: `module: short description` and follows [good practice](https://chris.beams.io/posts/git-commit/)
* [ ] Passes whitespace checkers
* [ ] Passes warning tests
* [ ] Passes all tests
* [ ] Add comments such that someone without knowledge of the code could understand
* [ ] Add Devel Docs in the `doc/` directory for any new code design

### CODE REVIEW

File: mpich-EL_7.spec [hunk 1]
```diff
@@ -325,8 +324,6 @@ make testing
 %{_libdir}/%{name}/lib/pkgconfig/
 %{_libdir}/%{name}/lib/*.so
 %{_libdir}/%{name}/bin/mpicc
-%{_libdir}/%{name}/bin/mpic++
```
Comment (mchaarawi) [inline mpich-EL_7.spec:322]: if you re-enable the c++ bindings, do you need to remove these?

Comment (brianjmurrell) [reply mpich-EL_7.spec]: Yeah.  b1c01c5.

So we are getting the full build, so all of that is working.  It's just working if and how `make testing` can be made to work.  It seems it was written to assume `make install` had already been run.  Which it has technically in the RPM build, but it was run with a `$DESTDIR`.

**Comment by brianjmurrell** [comment]
Oh.  I was going to clean this up before we landed it.  There were a lot of little debugging commits and so forth that needed squashing down into one commit.

**Comment by mchaarawi** [comment]
> Oh. I was going to clean this up before we landed it. There were a lot of little debugging commits and so forth that needed squashing down into one commit.
> 
> Oh, right. There are a couple of other patches that need to land to daos also before this will build successfully on the daos_adio-rpm branch.

so would you rather i revert it and restore the branch, or create a new PR?

**Comment by brianjmurrell** [comment]
Let's just leave it be.  Once the DAOS PRs land I will just do a _Build Now_ on mpich to get a passing build.

**Pull request merged**